.chloggen/admission-control-for-otlp.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
It might be good to call out exactly which configuration options have been moved in this changelog entry. This would include where the original config layout and the new location. Something like `arrow.waiter_limit` -> `admission.waiter_limit`, etc.

Reply:
Added to the changelog note!